cli/src/commands/config/path.rs

```diff
@@ -40,13 +39,7 @@ pub fn cmd_config_path(
     args: &ConfigPathArgs,
 ) -> Result<(), CommandError> {
     for config_path in args.level.config_paths(command.config_env())? {
-        writeln!(
-            ui.stdout(),
-            "{}",
-            config_path
-                .to_str()
-                .ok_or_else(|| user_error("The config path is not valid UTF-8"))?
-        )?;
+        writeln!(ui.stdout(), "{}", config_path.display())?;
```

Comment:
revert (for the same reason as `workspace root`)

Reply:
Done